### ISSUE TITLE
Now  default file encording is not utf-8?

### ISSUE THREAD

**Issue by totodo713** [ruby-formatter/rufo#197]
hi,

I use rufo 0.7.0 on ruby ruby 2.6.5p114  within windows 10.
Rufo is not run, because multibytes charactor is in file.
Rufo is done formatting, when I wrote Magic word (`# encoding: utf-8`).

I don't hope to wrote it, if Rufo's default file encording is not UTF-8.
Please fix it.

thanx.

**Comment by gingermusketeer** [comment]
@totodo713 Could you include a copy of the error you are getting when the magic comment is not present? If you could attach the file you are having a problem with that would be helpful as well.

**Comment by gingermusketeer** [comment]
@totodo713 I am going to close this for now but please reopen with the above information if you have a moment to upload it.

**Comment by totodo713** [comment]
@gingermusketeer sorry, I'm too lately. Is my answer OK ?

Before the first :
![ Before The first](https://user-images.githubusercontent.com/35787394/72689682-56a3f280-3b57-11ea-913e-401830937073.png)

After the first  on errored:
![After the first on errored](https://user-images.githubusercontent.com/35787394/72689693-73d8c100-3b57-11ea-8c57-11839c864f59.png)

Before the second with the Magic word
![before second with the Magic word](https://user-images.githubusercontent.com/35787394/72689706-a1256f00-3b57-11ea-8831-c3e65e0f5a41.png)

After the second on succeeded: 
![After the second on succeeded](https://user-images.githubusercontent.com/35787394/72689721-c0bc9780-3b57-11ea-8f2d-1458d6171c9a.png)

my source file here:

```ruby
# encoding: utf-8
# frozen_string_literal: true

class StockManagement < ApplicationRecord
  belongs_to :paper_yard
  SUCCESS_CODE = 0
  SUCCESS_CODE.freeze
  ERROR_CODE = 99
  ERROR_CODE.freeze

  validates :item_id, presence: true, length: { maximum: 10 }
  validates :item_name, presence: true, length: { maximum: 30 }
  validates :paper_yard_id, presence: true, length: { maximum: 10 }
  validates :paper_yard_name, presence: true, length: { maximum: 50 }
  validates :target_year_month, presence: true, length: { maximum: 6 }
  validates :collect_amount, presence: true, numericality: { only_integer: true }, length: { maximum: 9 }
  validates :delivery_amount, presence: true, numericality: { only_integer: true }, length: { maximum: 6 }
  validates :stock_amount, presence: true, numericality: { only_integer: true }, length: { maximum: 8 }

  class << self
    def index(_client_name, _item_id, _now_year_month, _until_year_month, _current_user)
      result[:message] = "コントラスト"
      result
    end
  end
end

```

**Comment by totodo713** [comment]
@gingermusketeer I guess I cannot reopen this issue because collaborator closed.

**Comment by gingermusketeer** [comment]
@totodo713 This should be resolved in 0.12 which has just been released. Would you be able to test it and make sure it is working for you?

**Comment by totodo713** [comment]
@gingermusketeer Thanks! I get to working sure!
